.github/workflows/master.yml

```diff
@@ -19,6 +19,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      # vale does not support pattern to ignore folders
+      # therefore it is necessary to use it before node_modules are installed
+      - name: documentation quality check
```

Comment:
I wonder if we want this check to be ran on pull request instead? To make sure the changed content by vale linter can be reviewed before merge!

Reply:
and another thing, is it possible to config this action to only be ran when documentation is changed?

Reply:
We can simply copy paste it in this (file)[https://github.com/spotify/backstage/blob/master/.github/workflows/ci.yml]. But this wont work when there is update in ```microsite/**``` as it is ignored.

We have to add separate job for that.

Any suggestion on what to do further?

Reply:
Ah I see, I think the solution then would be to create a separate ci workflow to run vale linter! To cover the microsite changes as well!